apps/aecore/test/aecore_txs_SUITE.erl

```diff
@@ -93,17 +93,14 @@ txs_gc(Config) ->
     Height1 = Height0 + length(Blocks1), %% Very unlikely to be > 6...
 
     %% At Height1 there should be 4 transactions in mempool
-    case Height1 of
-        2                -> pool_check(N1, 4);
-        HH1 when HH1 > 2 -> ct:log("Skipping assert; micro fork advanced height to far...")
-    end,
+    pool_check(N1, 4, {Height1, 2}),
 
     %% Mine 1 more key block to ensure one TX is GC:ed.
     aecore_suite_utils:mine_key_blocks(N1, 1),
     Height2 = Height1 + 1,
 
     %% Now at Height2 there should be 3 transactions in mempool - _GC1 is GC:ed
-    pool_check(N1, 3),
+    pool_check(N1, 3, {Height2, 6}),
```

Comment:
I understand this is main change.

Reply:
The TX is GC:ed at 7, so 6 is the highest height were we can see this effect.